README.md

```diff
@@ -180,3 +196,15 @@ scripts if requested.
 
 There are several tools trying to solve similar problems as Jujutsu. See
 [related work](https://martinvonz.github.io/jj/latest/related-work) for details.
+
+## Mandatory Google Disclaimer
+
+I (Martin von Zweigbergk <martinvonz@google.com>) started Jujutsu as a hobby
+project in late 2019, and it has evolved into my full-time project at Google,
+with several other Googlers assisting development. That said, **this is not a
+Google product**; and outside contributors are very welcome, too!
```

Comment:
Can you split these two sentences so it doesn't sound like "contributions are welcome since this is not a Google product". The point with "This is not a Google product" is rather to say that Google doesn't provide any support for it. Actually, the bit about "contributions are welcome" might be nice to include somewhere else (maybe with a link to contributing.md), unrelated to the disclaimer. But I don't mind if you just delete that bit if you don't find a good place for it.

Reply:
In fact, I'd put the two thoughts into separate paragraphs. For example:

> I (Martin von Zweigbergk <martinvonz@google.com>) started Jujutsu as a hobby
project in late 2019, and it has evolved into my full-time project at Google.
That said, **this is not a Google product**.
>
> We have both Googlers and non-Googlers assisting development; everyone is welcome!

I'm also itching to add

> We do have a few [policies and suggestions](https://martinvonz.github.io/jj/prerelease/contributing/#policies) for contributors.

or something along those lines so that people aren't surprised by the CLA, but that could also scare people away, I'm unsure.

Reply:
Minor point: you should add a comma (like I added above) or some other separator between Martin's name and email, otherwise it looks weird rendered. I noticed this when writing the previous comment.

Reply:
Gave it another round and split up this section based on your feedback. Let me know what you think!